cli/internal/ffi/ffi.go

```diff
@@ -42,6 +52,16 @@ func Marshal[M proto.Message](c M) C.Buffer {
 	return toBuffer(bytes)
```

Comment:
Similar to above, can you document what happens when Go frees the slice that allocated the byte buffer?